src/rpdk/core/contract/suite/handler_delete.py

```diff
@@ -24,12 +25,13 @@
 
 @pytest.fixture(scope="module")
 def deleted_resource(resource_client):
-    request = model = resource_client.generate_create_example()
+    request = input_model = model = resource_client.generate_create_example()
     try:
         _status, response, _error = resource_client.call_and_assert(
             Action.CREATE, OperationStatus.SUCCESS, request
         )
         model = response["resourceModel"]
+        test_input_equals_output(resource_client, input_model, model)
```

Comment:
This is testing the create input and output, which it's covered in the create contract test.  It would be noise for the delete contract test. E.g. the delete contract should not fail because create input/output fails.

Reply:
I wanted to ensure we had consistent behavior check across all contract test otherwise, the downstream teams usually ask why some creates are passing vs others.